.npmrc

```diff
@@ -1,5 +1,3 @@
-# Install peers for convenience
-auto-install-peers=true
```

Comment:
For future reviewers, it seems like the default for `auto-install-peers` is `true`, so it should be fine to remove here: https://pnpm.io/npmrc#auto-install-peers